<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30)</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>790</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Â© Created by Krylov</string>
   </property>
  </widget>
  <widget class="QLabel" name="standard_welcome">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>LOGGED IN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="logged_in_username">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>15</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 255, 127);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>740</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 87 18pt &quot;Arial Black&quot;;
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_image_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>user.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>DB: GuideCompanies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exploreButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 221, 255);
font: 87 14pt &quot;Arial Black&quot;;
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>EXPLORE</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tables_list">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>341</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>811</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="table_name_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Arial Black&quot;;
margin-left:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_permission">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>720</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>730</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 221, 255);
font: 87 14pt &quot;Arial Black&quot;;
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QComboBox" name="table_comboBox1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>730</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 20px;
	color: white;
	padding-left:20px;
	padding-right:20px;
	background-color: rgb(30, 30, 30);
	font-size: 18px;
}
QComboBox QAbstractItemView {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>770</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 221, 255);
font: 87 14pt &quot;Arial Black&quot;;
border-radius: 13px;</string>
   </property>
   <property name="text">
    <string>EDIT TABLE</string>
   </property>
  </widget>
  <widget class="QComboBox" name="table_comboBox2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>730</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 20px;
	color: white;
	padding-left:20px;
	padding-right:20px;
	background-color: rgb(30, 30, 30);
	font-size: 18px;
}
QComboBox QAbstractItemView {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="help_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>770</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="userRes.qrc"/>
 </resources>
 <connections/>
</ui>
